src/tests/stdlib/memmove.c

```diff
@@ -0,0 +1,59 @@
+#include <embox/test.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+EMBOX_TEST_SUITE("Test suite for memmove() function");
+
+
+TEST_CASE("move full data")
+{
+    const    char    src[]="string";
+    char    dest[sizeof(src)+1];
+
+    test_assert_not_null(dest) ;
+    memmove((void*)dest,(void*)src,(size_t)strlen(src));
```

Comment:
Too many casts

strlen returns size_t without casting
I think 
`memmove(dest, src, strlen(src));`
will enough

Reply:
ohhh sorry , i have done mistake ; thank you